js/ui/lookup.js

```diff
@@ -963,6 +994,10 @@ const Lookup = DropDownList.inherit({
                 break;
             case 'cleanSearchOnOpening':
             case '_scrollToSelectedItemEnabled':
+            case 'disableOptionCentering':
+                if(this.option('_scrollToSelectedItemEnabled') && !value) {
```

Comment:
Is it OK that `cleanSearchOnOpening` and `_scrollToSelectedItemEnabled` will be handled in the same way as `disableOptionCentering`

Reply:
Fixed